willa/config/__init__.py

```diff
@@ -111,3 +115,8 @@ def get_model() -> BaseChatModel:
             temperature=float(CONFIG['CHAT_TEMPERATURE'])
         )
     return model
+
+def get_langfuse_client() -> Langfuse:
+    """Return a configured instance of the Langfuse client. Currently relies on
+    Langfuse's environment variables."""
+    return Langfuse()
```

Comment:
Leaving a comment here because I haven't written a note to myself (or a Jira ticket) yet: it would be *really* nice to call this with `release=git_or_docker_sha`.  That way we can easily tell if we have improved (or regressed) a feature with a specific change.

Reply:
i think we can inject that in the `metadata` here: https://github.com/BerkeleyLibrary/willa/pull/44/files#diff-43d4dad11aff240e3562958e68747e9b92319d6de04f3abf645742ed23136c72R45